op-node/rollup/derive/kromampt_upgrade_transactions.go

```diff
@@ -31,15 +36,21 @@ var (
 	updateSequencerFeeVaultProxySource = UpgradeDepositSource{Intent: "KromaMPT: Sequencer Fee Vault Proxy Update"}
 	deployL1BlockMPTSource             = UpgradeDepositSource{Intent: "KromaMPT: L1 Block Deployment"}
 	updateL1BlockMPTProxySource        = UpgradeDepositSource{Intent: "KromaMPT: L1 Block Proxy Update"}
+	deployGasPriceOracleMPTSource      = UpgradeDepositSource{Intent: "KromaMPT: Gas Price Oracle Deployment"}
+	updateGasPriceOracleMPTProxySource = UpgradeDepositSource{Intent: "KromaMPT: Gas Price Oracle Proxy Update"}
+	enableKromaMPTSource               = UpgradeDepositSource{Intent: "KromaMPT: Gas Price Oracle Set Kroma MPT"}
+
+	enableKromaMPTInput = crypto.Keccak256([]byte("setKromaMPT()"))[:4]
 
 	l1BlockMPTDeploymentBytecode        = common.FromHex(bindings.L1BlockMetaData.Bin)
 	sequencerFeeVaultDeploymentBytecode = common.FromHex(bindings.ProtocolVaultMetaData.Bin)
 	baseFeeVaultDeploymentBytecode      = common.FromHex(bindings.ProtocolVaultMetaData.Bin)
 	l1FeeVaultDeploymentBytecode        = common.FromHex(bindings.L1FeeVaultMetaData.Bin)
+	gasPriceOracleMPTDeploymentBytecode = common.FromHex(bindings.GasPriceOracleMetaData.Bin)
```

Comment:
Could you fix this to the actual bytecode? Relevant comment [here](https://github.com/kroma-network/kroma/pull/398#discussion_r1881229723).